src/client/slices/settings.ts

```diff
@@ -41,35 +41,34 @@ const settingsSlice = createSlice({
         [payload.key]: payload.value,
       },
     }),
-    togglePreviewMarkdown: state => ({
+    togglePreviewMarkdown: (state) => ({
       ...state,
       previewMarkdown: !state.previewMarkdown,
     }),
-    toggleDarkTheme: state => ({
+    toggleDarkTheme: (state) => ({
       ...state,
       darkTheme: !state.darkTheme,
     }),
-    toggleSidebarVisibility: state => ({
+    toggleSidebarVisibility: (state) => ({
       ...state,
       sidebarVisible: !state.sidebarVisible,
     }),
     updateNotesSortStrategy: (state, { payload }: PayloadAction<NotesSortKey>) => ({
       ...state,
       notesSortKey: payload,
     }),
-    loadSettings: state => ({
+    loadSettings: (state) => ({
       ...state,
       loading: true,
     }),
-    loadSettingsError: state => ({
+    loadSettingsError: (state) => ({
       ...state,
       loading: false,
     }),
     loadSettingsSuccess: (state, { payload }: PayloadAction<SettingsState>) => ({
       ...payload,
       isOpen: false,
       loading: false,
-      notesSortKey: !state.notesSortKey ? NotesSortKey.LAST_UPDATED : state.notesSortKey,
```

Comment:
I see we now have LAST_UPDATED key by default in initial state. So, when the app loads, sort key is already there. Then, load settings is called, and notesSortKey payload was getting overridden by this line. Is there something I'm missing here ?